ci/scripts/bump-and-tag.bash

```diff
@@ -38,6 +44,28 @@ mv ${package_tmp} ${package_json_path}
 
 git commit Cargo.toml ${plugin_toml_path} ${package_json_path} -m "chore: Bump version to $version"
 
+# Select all package dependencies that match $bump_deps_pattern and bump them to $bump_deps_version
+if [[ "$bump_deps_pattern" != '' ]]; then
+  deps=$(toml get Cargo.toml workspace.dependencies | jq -r "keys[] | select(test(\"$bump_deps_pattern\"))")
+  for dep in $deps; do
+    if [[ -n $bump_deps_version ]]; then
+      toml_set_in_place Cargo.toml "workspace.dependencies.$dep.version" "$bump_deps_version"
+    fi
+
+    if [[ -n $bump_deps_branch ]]; then
+      toml_set_in_place Cargo.toml "workspace.dependencies.$dep.branch" "$bump_deps_branch"
+    fi
```

Comment:
As mentioned on Discord, this needs update as well similar to how it's done for zenoh-c opaque-types: https://github.com/eclipse-zenoh/zenoh-c/blob/main/ci/scripts/bump-and-tag.bash#L76, so https://github.com/eclipse-zenoh/zenoh-ts/blob/main/zenoh-plugin-remote-api/Cargo.toml#L91 gets bumped too.

Reply:
Added update of `package.metadata.deb.depends`